src/app/network/network-list/network-list.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, EventEmitter, Input, Output } from "@angular/core";
+
+@Component({
+	selector: "network-list",
+	templateUrl: "network-list.component.html",
+})
+export class NetworkListComponent {
+	@Output()
+	addNetworkClick = new EventEmitter();
+
+	@Input()
+	networks = [];
```

Comment:
Same for other properties in this PR.

```suggestion
	public networks = [];
```